biodatasets/bionlp_st_2013_pc/bionlp_st_2013_pc.py

```diff
@@ -0,0 +1,234 @@
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import List
+
+import datasets
+
+from utils import parsing, schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_DATASETNAME = "bionlp_st_2013_pc"
+_UNIFIED_VIEW_NAME = "bigscience"
+
+_CITATION = """\
+@inproceedings{ohta-etal-2013-overview,
+    title = "Overview of the Pathway Curation ({PC}) task of {B}io{NLP} Shared Task 2013",
+    author = "Ohta, Tomoko  and
+      Pyysalo, Sampo  and
+      Rak, Rafal  and
+      Rowley, Andrew  and
+      Chun, Hong-Woo  and
+      Jung, Sung-Jae  and
+      Choi, Sung-Pil  and
+      Ananiadou, Sophia  and
+      Tsujii, Jun{'}ichi",
+    booktitle = "Proceedings of the {B}io{NLP} Shared Task 2013 Workshop",
+    month = aug,
+    year = "2013",
+    address = "Sofia, Bulgaria",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/W13-2009",
+    pages = "67--75",
+}
+"""
+
+_DESCRIPTION = """\
+the Pathway Curation (PC) task is a main event extraction task of the BioNLP shared task (ST) 2013.
+The PC task concerns the automatic extraction of biomolecular reactions from text.
+The task setting, representation and semantics are defined with respect to pathway
+model standards and ontologies (SBML, BioPAX, SBO) and documents selected by relevance
+to specific model reactions. Two BioNLP ST 2013 participants successfully completed
+the PC task. The highest achieved F-score, 52.8%, indicates that event extraction is
+a promising approach to supporting pathway curation efforts.
+"""
+
+_HOMEPAGE = "https://github.com/openbiocorpora/bionlp-st-2013-pc"
+
+_LICENSE = "CC BY-SA 3.0"
+
+_URLs = {
+    "bionlp_st_2013_pc": "https://github.com/openbiocorpora/bionlp-st-2013-pc/archive/refs/heads/master.zip",
+}
+
+_SUPPORTED_TASKS = [Tasks.EVENT_EXTRACTION]
+_SOURCE_VERSION = "1.0.0"
+_BIGBIO_VERSION = "1.0.0"
+
+
+class bionlp_st_2013_pc(datasets.GeneratorBasedBuilder):
+    """the Pathway Curation (PC) task is a main event extraction task of the BioNLP shared task (ST) 2013."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="bionlp_st_2013_pc_source",
+            version=SOURCE_VERSION,
+            description=_DESCRIPTION,
+            schema="source",
+        ),
+        BigBioConfig(
+            name="bionlp_st_2013_pc_bigbio_kb",
+            version=BIGBIO_VERSION,
+            description=_DESCRIPTION,
+            schema="bigbio_kb",
+            subset_id="bionlp_st_2013_pc",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = _DATASETNAME
```

Comment:
DEFAULT_CONFIG_NAME should be either of the name of the config defined as above (either "bionlp_st_2013_pc_source" or "bionlp_st_2013_pc_bigbio_kb"

Reply:
I've seen the defaults usually go to source.